app/model/authorisation.py

```diff
@@ -68,8 +68,8 @@ class AuthEndpoint(str, enum.Enum):
     },
     # Corpus
     AuthEndpoint.CORPUS: {
-        AuthOperation.CREATE: AuthAccess.ADMIN,
-        AuthOperation.READ: AuthAccess.ADMIN,
-        AuthOperation.UPDATE: AuthAccess.ADMIN,
+        AuthOperation.CREATE: AuthAccess.SUPER,
```

Comment:
In the future, the corpus routes should be available to admin service users rather than only superusers, but because we haven't fully fleshed out this functionality yet I think it's safer to restrict to superusers only for now.